Basic/APIDiorama/Assets/UseCaseSamples/RPCs/Scripts/MoodManager.cs

```diff
@@ -0,0 +1,83 @@
+using System.Collections;
+using UnityEngine;
+
+namespace Unity.Netcode.Samples.APIDiorama
+{
+    /// <summary>
+    /// Manages the mood of a player or NPC
+    /// </summary>
+    public class MoodManager : NetworkBehaviour
+    {
+        [SerializeField] SpeechBubble m_SpeechBubblePrefab;
+        SpeechBubble m_SpeechBubble;
+
+        [SerializeField, Tooltip("The seconds that will elapse between data changes"), Range(2, 5)]
+        float m_SecondsBetweenDataChanges;
+        float m_ElapsedSecondsSinceLastChange;
+
+        readonly string[] s_ChatMessages = new string[]
+        {
+            "Have a lovely day",
+            "Are you pineapple? Duck you, potato!",
+            "Today I feel like pineapple!",
+            "Wow you're awesome!"
+        };
+
+        void Update()
+        {
+            if (!IsOwner)
+            {
+                //we don't want to send chat messages from other players!
+                return;
+            }
+
+            m_ElapsedSecondsSinceLastChange += Time.deltaTime;
+            if (m_ElapsedSecondsSinceLastChange >= m_SecondsBetweenDataChanges)
+            {
+                m_ElapsedSecondsSinceLastChange = 0;
+                OnServerMoodMessageReceivedServerRpc(s_ChatMessages[Random.Range(0, s_ChatMessages.Length)]);
+            }
+        }
+
+        [ServerRpc(RequireOwnership = true)]
```

Comment:
This is the default behaviour, any reason why we're setting it here?
The act of setting it tells the reader there's something to look at and be careful about.
You could instead have a comment saying "this is owner only by default"

Reply:
Fixed